xshop/users/serializers.py

```diff
@@ -3,33 +3,35 @@
 from django.contrib import auth
 from rest_framework.exceptions import AuthenticationFailed
 
+
 class UserSerializer(serializers.ModelSerializer):
     class Meta:
         model = User
-        fields = ('mobile', 'password' )
+        fields = ("mobile", "password")
+
 
 class LoginSerializer(serializers.ModelSerializer):
     mobile = serializers.CharField(max_length=11)
     password = serializers.CharField(max_length=25)
 
     class Meta:
         model = User
-        fields = ['mobile','password']
+        fields = ["mobile", "password"]
 
     def validate(self, attrs):
-        mobile = attrs.get('mobile', '')
-        passward = attrs.get('password', '')
+        mobile = attrs.get("mobile", "")
+        password = attrs.get("password", "")
```

Comment:
`attrs.get("mobile")` instead of `attrs.get("mobile", "")` the default is None, don't add a default, not necessary
same for password